WebHostLib/static/assets/tutorial/checksfinder/checksfinder_en.md

```diff
@@ -0,0 +1,47 @@
+# ChecksFinder Randomizer Setup Guide
+
+## Required Software
+
+- ChecksFinder from
+  my [Github releases Page for the game](https://github.com/jonloveslegos/ChecksFinder/releases) (latest version)
```

Comment:
```suggestion
  the [Github releases Page for the game](https://github.com/jonloveslegos/ChecksFinder/releases) (latest version)
```

Small nitpick, but probably those reading this guide don't know who 'my' is, even if it's obvious by visiting the github repo.